README.md

```diff
@@ -35,3 +35,11 @@ npm run codegen
 # Build .js/.d.ts files
 npm run build
 ```
+
+NOTE:
+for now because slinky/oracle/v1/query.proto has a proto syntax error types have some syntax that codegen cannot recognize,
+flow is like this:
+- yarn run codegen;
+- remove precodegen command from package.json;
+- remove failing `option (google.api.http)` from protos/neutron/slinky/oracle/v1/query.proto;
+- yarn run codegen.
```

Comment:
Can you make all of this automated?

Reply:
i think we can fix slinky tbh, since it's in our codebase anyway

Reply:
fixed